encoders/src/main/scala/au/csiro/pathling/encoders/datatypes/DecimalCustomCoder.scala

```diff
@@ -87,5 +87,5 @@ object DecimalCustomCoder {
 
   val scale: Int = 6
   val precision: Int = 26
-  val decimalType = DataTypes.createDecimalType(precision, scale)
+  val decimalType: types.DecimalType = DataTypes.createDecimalType(precision, scale)
 }
```

Comment:
Does the comment here need updating, is it actually DECIMAL(26,6)?

I'll add the precision and scale to the documentation, so that users know what the limits are.